highcharts_core/js_literal_functions.py

```diff
@@ -581,6 +581,25 @@ def convert_js_property_to_python(property_definition, original_str = None):
     elif property_definition.value.type == 'ClassExpression':
         return JavaScriptClass._convert_from_js_ast(property_definition.value,
                                                     original_str)
+    elif property_definition.value.type == 'BinaryExpression':
+        property_value = property_definition.value
+        operator = property_value.operator
+        left, right = property_value.left, property_value.right
+        left_type, right_type = left.type, right.type
+        
+        if (left_type not in ['Literal']) or (right_type not in ['Literal']):
+            raise errors.HighchartsParseError(f'unable to find two Literal values within'
+                                              f'a Binary expression. Found: '
+                                              f'{left_type, right_type}')
+
+        left_value, right_value = left.value, right.value
+        if operator not in ['+', '-', '/']:
+            raise errors.HighchartsParseError(f'operator "{operator}" not supported within '
+                                              f'Binary expression parsing')
+        left_value = validators.string(left_value, allow_empty = False)
+        right_value = validators.string(right_value, allow_empty = False)
+        
+        return left_value + right_value
```

Comment:
Whatever the operator `+`, `-`, `/` in the expression, you apply the `+` operation, it's super confusing.

Shouldn't `+` be the only operator accepted on strings ? 
`'hello' - 'bye'` and `'hello' / 'bye'` are both unsupported operand on str in Python.